packages/web/src/app/meeting/agenda/add/page.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
+
+import FlexWrapper from "@sparcs-clubs/web/common/components/FlexWrapper";
+import PageHead from "@sparcs-clubs/web/common/components/PageHead";
+import NewAgendaFrame from "@sparcs-clubs/web/features/meeting/agenda/add/frames/NewAgendaFrame";
+
+const AgendaAddPage: React.FC = () => (
```

Comment:
`Page`는 다들 굳이 안 붙이는듯..??

```suggestion
const AgendaCreate: React.FC = () => (
```